quick-start/src/main/ui/app/jobs/job-export.component.ts

```diff
@@ -1,36 +1,28 @@
-import {Component, HostListener, Inject, Input} from '@angular/core';
-import {MdlDialogReference, MdlDialogService} from '@angular-mdl/core';
+import {Component, Inject} from '@angular/core';
 import {JobService} from "./jobs.service";
+import {MdlDialogService} from '@angular-mdl/core';
 
 @Component({
   selector: 'job-export-dialog',
-  templateUrl: 'job-export.component.html'
+  template: `
+  <app-job-export-ui      
+    [jobIds]="jobIds"
+    (exportClicked)="export()"
+  ></app-job-export-ui>
+  `
 })
 export class JobExportDialogComponent {
-
   jobIds: string[];
-  question: string;
 
   constructor(
-    public dialog: MdlDialogReference,
-    private dialogService: MdlDialogService,
     private jobService: JobService,
+    private dialogService: MdlDialogService,
     @Inject('jobIds') jobIds: string[]
   ) {
-
     this.jobIds = jobIds;
-    this.question = "Export and download ";
-    if (jobIds.length === 0) {
-      this.question += "all jobs and their traces?";
-    } else if (this.jobIds.length === 1) {
-      this.question += "1 job and its traces?";
-    } else {
-      this.question += this.jobIds.length + " jobs and their traces?";
-    }
   }
 
   public export() {
-    this.dialog.hide();
     this.jobService.exportJobs(this.jobIds)
       .subscribe(response => {
           let body = response['_body'];
```

Comment:
@alexsmr Here's the block where the generic call to the dialogService.alert() is called.